src/App.tsx

```diff
@@ -2,108 +2,38 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { Posts } from './types/Posts';
+import { Comments } from './types/Comments';
+import { Users } from './types/Users';
+import { FullPost } from './types/FullPosts';
+
+import { PostList } from './components/PostList';
+
+const preparePost = (
+  posts: Posts[],
+  comments: Comments[],
+  users: Users[],
+): FullPost[] => {
+  return posts.map(post => ({
+    ...post,
+    comments: comments.filter((comment) => comment.postId === post.id),
+    user: users.find((user) => user.id === post.userId) || null,
```

Comment:
you can create separate functions for this logic 
`getUserById` & `getCommentsById`